tournaments/index.js

```diff
@@ -1042,7 +1042,7 @@ let commands = {
 			if (tournament.banlist.length < 1) {
 				return this.errorReply("The tournament's banlist is empty.");
 			}
-			this.sendReplyBox("This tournament includes:<br />" + tournament.getBanlist());
+			this.sendReplyBox('<div class="infobox infobox-limited">This tournament includes:<br />' + tournament.getBanlist() + '</div>');
```

Comment:
To clarify here, probably something like this would be better:
``this.sendReply(`|raw|<div class="infobox infobox-limited">This tournament includes:<br />${tournament.getBanlist()}</div>`);``